docs/examples/flex/flexItem.js

```diff
@@ -0,0 +1,19 @@
+// @flow strict
+import { type Node } from 'react';
+import { Flex, Box, Button } from 'gestalt';
+
+export default function Example(): Node {
+  return (
+    <Flex width="100%" height="100%" justifyContent="center" alignItems="center">
+      <Box borderStyle="sm" paddingX={2} paddingY={3} rounding={3} width="100%">
```

Comment:
The layout on this one is a bit funky now, maybe remove the border on this Box

Reply:
DONE